W-14164266-small-fixes-duke (#194)
mulesoft/docs-code-builder: modules/ROOT/pages/troubleshoot-loading-errors.adoc

== Loading Error in the Application UI

A *Loading...* error in the UI canvas often occurs under these conditions:

* When importing a project due to a missing dependency 
+
To check for a missing dependency:

//step open output panel
include::partial$acb-reusable-steps.adoc[tags="open-output-panel"]
. Select "Mule DX Server" from the drop-down menu.
. Search for an error caused by the missing dependency, similar to this one:
+
----
Caused by: org.eclipse.aether.transfer.ArtifactNotFoundException: 
Failure to find org.mule.examples:accelerator-common-config-module:jar:mule-plugin:1.1.0 
in https://repo.maven.apache.org/maven2/ was cached in the local repository, 
resolution will not be reattempted until the update interval of central has 
elapsed or updates are forced
----
. If you find a missing dependency, log in to Anypoint Platform.
+
The canvas UI loads after all dependencies load.

* When a browser extension is interfering with the UI loading process  
+

> image::troubleshoot-loading-error.png["Loading Error in the UI"]

Reviewer: ```suggestion
image::troubleshoot-loading-error.png["Loading error in the UI"]
```

== Loading Error in the Application UI

> A *Loading...* error in the UI can occur when:

Author: ```suggestion
A *Loading...* error in the UI can occur under these conditions:
```

> * When importing a project due to a missing dependency 

Author: ```suggestion
* Importing a project due to a missing dependency 
```

+
To check for a missing dependency:

//step open output panel
include::partial$acb-reusable-steps.adoc[tags="open-output-panel"]
. Select "Mule DX Server" from the drop-down menu.
. Search for the error caused by the missing dependency, similar to this one:
+
----
Caused by: org.eclipse.aether.transfer.ArtifactNotFoundException: 
Failure to find org.mule.examples:accelerator-common-config-module:jar:mule-plugin:1.1.0 
in https://repo.maven.apache.org/maven2/ was cached in the local repository, 
resolution will not be reattempted until the update interval of central has 
elapsed or updates are forced
----
. If you find a missing dependency, log in to Anypoint Platform.
+
The canvas UI loads after all dependencies load.

* When a browser extension is interfering with the UI loading process  
+
image::troubleshoot-loading-error.png["Loading Error in the UI"]
+
To address this issue, try:
+
* Disabling browser extensions and reloading the application 
* Using another browser that is not using extensions

> If the canvas UI does not open (the load state image keeps showing) after importing an existing project, it may be caused by a missing dependency. Follow these steps:

Author: ```suggestion
If the canvas UI does not open but appears to be loading after you import an existing project, the issue might be due to a missing dependency. Follow these steps:
```

> 

Author: TODO: EDIT AND MAKE SURE INFO IS COVERED IN THE CORRECT PLACE
